shared/chat/conversation/container.js

```diff
@@ -14,27 +15,46 @@ import type {Props} from '.'
 type OwnProps = {}
 type State = {
   sidePanelOpen: boolean,
+  inputText: string,
 }
 
 class ConversationContainer extends Component<void, Props, State> {
   state: State
+  _inputTextMap: {[key: string]: ?string}= {}
```

Comment:
Hm, is the TODO something extra you want to do as part of this PR, or to happen after it's merged?  It won't be seen after this goes in, so it seems like it'd make more sense to leave the TODO as a comment in the source here, or create a JIRA ticket for it.